usaspending_api/references/management/commands/load_gtas.py

```diff
@@ -108,7 +116,10 @@ def column_statements(self):
         return "\n".join(simple_fields + inverted_fields + year_specific_fields)
 
     def tas_fk_sql(self):
-        return """UPDATE gtas_sf133_balances
-                    SET treasury_account_identifier = tas.treasury_account_identifier
-                    FROM treasury_appropriation_account tas
-                    WHERE tas.tas_rendering_label = gtas_sf133_balances.tas_rendering_label;"""
+        return """
+            UPDATE gtas_sf133_balances
+            SET treasury_account_identifier = tas.treasury_account_identifier
+            FROM treasury_appropriation_account tas
+            WHERE
+                tas.tas_rendering_label = gtas_sf133_balances.tas_rendering_label
+                AND gtas_sf133_balances.treasury_account_identifier IS DISTINCT FROM tas.treasury_account_identifier"""
```

Comment:
Why is this additional clause being added? 
`AND gtas_sf133_balances.treasury_account_identifier IS DISTINCT FROM tas.treasury_account_identifier`

Reply:
At first I was confused which table was being updated. Ultimately it doesn't impact anything. But it is a better SQL in case it is used separate from the GTAS ETL or is the ETL is refactored away from the purge and reinsert process. If this wasn't a hotfix I would have changed the GTAS script to add/delete/update instead of deleting everything and re-inserting since a majority of the records are pretty static

Reply:
Makes sense